src/io/flutter/run/LaunchState.java

```diff
@@ -90,6 +90,12 @@ public LaunchState(@NotNull ExecutionEnvironment env,
     DaemonConsoleView.install(this, env, workDir);
   }
 
+  @NotNull
+  protected Callback getCallback() {
```

Comment:
I know this isn't part of your PR, but given that we're increasing the access level here, consider renameing the `Callback` interface to something more qualified - `LaunchStateCallback`, `ConnectionCompleteCallback`, ...

Reply:
That ended up changing a bunch of identifiers but I think it is more readable now.